File: WinFormsAppProyecto/Modelos/Gasto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Modelos
{
    public class Gasto
    {
        public int id {  get; set; }
        public string concepto { get; set; }
        public double valor { get; set; }
        public Inquilino inquilino { get; set; }
        public Piso piso { get; set; }
    }
}

File: AndroidProyecto/app/src/main/java/com/example/androidappproyecto/modelos/Gasto.kt
package com.example.androidappproyecto.modelos

data class Gasto(
    val id: Int = 0,
    val concepto: String? = null,
    val valor: Double = 0.0,

    // Relaciones
    val inquilino: Inquilino? = null,
    val piso: Piso? = null
)